src/components/Signup.tsx

```diff
@@ -0,0 +1,41 @@
+import { auth } from 'config/firebase';
+
+import React, { useRef } from 'react';
```

Comment:
React isn't use in the code co it can be remove from inport

Reply:
fixed